content/blog/2025/anchor-position-area.md

```diff
@@ -0,0 +1,170 @@
+  alt: >
+    A hand with painted nails
+    placing a white square of paper
+    into a 9 by 9 grid.
+author: james
+sponsors: true
+tags:
+  - Article
+  - Anchor Positioning
+  - CSS
+summary: |
+  `position-area` might be my favorite part of the CSS Anchor Positioning spec,
+  with a ton of features packed in to make things just... work.
+  But there's no magic here, just a few key parts that work well.
+---
+
+{% import 'embed.macros.njk' as embed %}
+
+{% callout 'note', false %}
+
+The examples in this article are best viewed in a Chromium browser, version 131
+or later.
+
+{% endcallout %}
+
+On the surface, `position-area` seems like just a shorthand.
+Just replace `top: anchor(bottom); left: anchor(right)`
+with `position-area: bottom right` and it works.
+
+Or go even further, and replace that with `position-area: end`.
+
+{{ embed.codepen(
+  id='zxOVXOm',
+  title='position-area containing block',
+  user='jamessw',
+  height=300,
+  tab='css,result'
+) }}
+
+The idea behind `position-area` is that in most cases, you want your positioned
+element to be touching a specific corner or edge of the anchor. `position-area`
+creates a grid around the anchor, and lets you place your positioned element
+within that grid.
+
+[Anchor-tool.com](https://anchor-tool.com/) is a fantastic tool for exploring
+the different values for positioning an element within that grid.
+
+But `position-area` is more than a shorthand for `anchor()`, introducing some
+behaviors that may be surprising.
+
+## A new containing block
+
+One of the key differences between using `anchor()` and `position-area` is that
+`position-area` creates a new containing block for the positioned element. While
+we're used to dealing with containing blocks when positioning `position:
+absolute` inside of a `position: relative` ancestor, `position-area` creates a
+containing block that does not map to an element.
+
+This containing block is generated by the grid area that you select using
+`position-area`. For instance, with `position-area: top left`, the new
+containing block starts at the top left corner of the anchor's containing block
+and extends to the top left corner of the anchor itself.
+
+{{ embed.img(
+  src='blog/2025/anchor-area-containing-block.png',
+  alt='A 9-space grid, like a tic-tac-toe board, with an Anchor in the center square, and a square within the top left square for a new containing block.',
+  attrs={
+    'class': 'align-center',
+    'width': '1089',
+    'height': '997'
+  }
+) }}
+
+Note that this is not the positioned element's box, but rather the containing block _for_ the positioned element's box.
+
+## Percentages
+
+One of the ways we observe a containing block is that percentages are resolved
+in relation to the containing block. That means a `padding: 5%` on an element
+positioned using `position-area` will have a padding of 5% of the size of the
+containing block, the section of the grid that was selected.
+
+{{ embed.codepen(
```

Comment:
Helps if these two boxes align and have the same size. I think you can achieve the former by giving the anchors an explicit width. I'm not sure how to make sure they both stretch the same amount. 

Another way to more explicitly visualize the padding area is to layer two backgrounds, and then use background-clip so that the top one is clipped at the content-box.